rero_ils/permissions.py

```diff
@@ -86,6 +86,7 @@ def can_delete_organisation_records_factory(record, *args, **kwargs):
 
     user must have librarian or system_librarian role.
     returns False if a librarian tries to delete a system_librarian.
+    false if librarian tries to delete a librarian from another library.
```

Comment:
can be rewords as: "returns False if a librarian tries to delete a system_librarian and if librarian tries to delete a librarian from another library."